wremnants/histmaker_tools.py

```diff
@@ -13,7 +13,7 @@ def scale_to_data(result_dict, data_name = "dataPostVFP"):
     else:
         lumi = 1
 
-    logger.debug(f"Scale histograms with lumi={lumi}")
+    logger.warning(f"Scale histograms with luminosity = {lumi} /fb")
```

Comment:
You couldn't event fathom the amount of times I got histograms normalized to 1/fb without noticing because I was not using them for plots with data, I juts want it to always shine  ;-)
Info could be good too but would be less visible, definitely it must not be debug

Reply:
Yes I'd print the warning in the case that there's no data